DZ_3/forms.py

```diff
@@ -0,0 +1,13 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField, IntegerField, SelectField
+from wtforms.validators import DataRequired, EqualTo, Email, Length
+
+from flask_sqlalchemy import SQLAlchemy
+
+db = SQLAlchemy()
+class RegistrationForm(FlaskForm):
+    username = StringField('Username', validators=[DataRequired()])
+    email = StringField('Email', validators=[DataRequired(), Email()])
```

Comment:
Тут можно использовать `EmailField`, хоть и не обязательно, так как валидацией будут заниматься валидаторы
```suggestion
    email = EmailField('Email', validators=[DataRequired(), Email()])
```